engine/table/src/main/java/io/deephaven/engine/table/impl/multijoin/IncrementalMultiJoinStateManagerTypedBase.java

```diff
@@ -442,19 +439,14 @@ public boolean doRehash(boolean fullRehash, MutableInt rehashCredits, int nextCh
 
         Assert.eqZero(rehashPointer, "rehashPointer");
 
-        for (int ii = 0; ii < mainKeySources.length; ++ii) {
-            alternateKeySources[ii] = mainKeySources[ii];
-            mainKeySources[ii] = InMemoryColumnSource.getImmutableMemoryColumnSource(tableSize,
-                    alternateKeySources[ii].getType(), alternateKeySources[ii].getComponentType());
-            mainKeySources[ii].ensureCapacity(tableSize);
-        }
         alternateTableSize = oldTableSize;
```

Comment:
Does this assignment effect anything? I think it's immaterial, since we'll ignore any hashed values that are greater than the rehashPointer (which is zero). Some part of me wants to assign it to oldTableSize in newAlternate, and back to 1 in clearAlternate, just for cleanliness.

Reply:
I think what you have done is good.

Reply:
Although I don't love that you're clearing it in clearAlternate, and assigning it *after* newAlternate, I think this is good enough for now.